### PR TITLE
Fixed network tab 

### PR DESCRIPTION
**Pull request by arandomtim**
Fixed the network tab table colors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +19,8 @@ Type the following commands into SSH, line by line.
 
 ```bash
 cd /var/www/html/admin/style/vendor/
-sudo git clone https://github.com/thomasbnt/Night_Pihole.git
-cd Night_Pihole
+sudo git clone https://github.com/thomasbnt/Night_Pi-hole.git
+cd Night_Pi-hole
```
Comment (thomasbnt) [inline README.md]: Why do you rename this folder?

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Can you delete the Speedtest part? It's an extention to Pi-hole, and the repository is only for the dark theme.

File: README.md (hunk 1 above)
Comment (arandomtim) [reply README.md]: I’ve deleted the Speedtest parts. That and the renaming was me being too keen and ‘excited’ for this. (Please forgive me, I’m pretty new to this GitHub thing.)

Comment (thomasbnt) [reply README.md]: No problem and don't worry! Welcome on GitHub and thanks for contribute at this project!

**Review by thomasbnt — APPROVED** [review]

**Comment by thomasbnt** [comment]
PR merged! 

Thank you @arandomtim for contributed !

**Pull request merged**